pkg/tests/transpiled_examples/resource-config-pp/resource-config.pp

```diff
@@ -0,0 +1,24 @@
+config inputs {
```

Comment:
Is this used anywhere?

Reply:
I meant in general, this whole file?  But I see now that it's used for the `example` added above?